### PR TITLE
REL: Merging develop into master with elliptical fins addition

### PR DESCRIPTION
**Pull request by Gui-FernandesBR**
## Pull request type

Please check the type of change your PR introduces:

- [X] ReadMe, Docs and GitHub maintenance

## Pull request checklist

- ReadMe, Docs and GitHub maintenance:

  - [X] Spelling has been verified
  - [X] Code docs are working correctly 

## What is the new behavior?

I'm adding develop new Elliptical Fins to master branch so we can relase a new version of rocketpy project!

## Does this introduce a breaking change?

- [X] No

## Other information

Although conda forge isn't updated to v0.11.1 yet, that will not conflict with this new release.

### CODE REVIEW

**Review by MateusStano — APPROVED** [review]

**Comment by giovaniceotto** [comment]
Please wait until the end of the day before merging.

File: README.md
Comment (giovaniceotto) [inline README.md:308]: This does not seem right.

**Review by giovaniceotto — CHANGES_REQUESTED** [review]

File: README.md
Comment (Gui-FernandesBR) [reply README.md]: I'm sorry, I've used a simple "replace all" and did not expect these behavior. I fixed now.

**Comment by Gui-FernandesBR** [comment]
Everything seems good to me. @giovaniceotto any other problem or can we move forward?

**Review by giovaniceotto — CHANGES_REQUESTED** [review]
There is a major issue with the Getting Started.html file for Matlab. Its rendering is completely bugged. I have spent a couple of hours trying to fix it with no success. This cannot be approved until this is fixed.

An image for perspective:
image

**Comment by giovaniceotto** [comment]
@Gui-FernandesBR, please review my changes, approve them if everything is all right and merge.

**Review by giovaniceotto — APPROVED** [review]

**Comment by Gui-FernandesBR** [comment]
@giovaniceotto I'm still looking for an answer on the MATLAB problem.

Meanwhile, is it possible to create tests for documentation or at list the matlab integration?

**Comment by giovaniceotto** [comment]
All reviewers have approved, and all tests have passed. Time to merge. Then, it's time to create a new release. @Gui-FernandesBR should the general team or the outreach team do it?

**Comment by giovaniceotto** [comment]
@Gui-FernandesBR, I have fixed the MATLAB documentation problem this morning in this commit: https://github.com/RocketPy-Team/RocketPy/pull/225/commits/68a5cfbdcdfcf32ffb878346e2edd82a8874aac3

No, there is no way to implement a test for the MATLAB integration as that would require a license.

No, there is no straightforward way to create a test to check if the MATLAB getting started html is formatted correctly, unfortunately.

But all is good for now!

**Pull request merged**
